front/src/components/nft-card-information/NftCardInformation.tsx

```diff
@@ -10,23 +11,54 @@ const NftCardInformation = ({ data }: NftCardInformationProps) => {
   const { classes } = useStyles();
 
   return (
-    <div>
-      <GCText weight={500}>{data.name}</GCText>
-      <GCText weight={400} className={classes.priceText}>
-        Price: {data.mintingPrice}
+    <>
+      <GCText weight={700} size={32} color={"black"}>
+        {data.name}
       </GCText>
-      <div className={classes.listSize}>
-        <GCList size={12}>
-          <GCListItem>Rarity: {data.rarity}</GCListItem>
-          <GCListItem>Collection: {data.collection}</GCListItem>
-          <GCListItem>First owner: {data.firstOwner}</GCListItem>
-          <GCListItem>Creation date: {data.mintingDate}</GCListItem>
-        </GCList>
+      <GCText weight={400} size={16} color={"black"} className={classes.collection}>
+        Collection:
+        <GCText size={16} weight={700} color={"darkviolet"}>
+          {data.collection}
+        </GCText>
+      </GCText>
+
+      <div className={classes.frame}>
+        <GCText size={16} color={"black"} weight={700} className={classes.frameTitle}>
+          <IconPhoto color={"darkviolet"} /> About the Build
+        </GCText>
+        <GCText size={16} color={"black"} weight={400} className={classes.frameText}>
+          {data.gunDescription}
+        </GCText>
       </div>
-      <div className={classes.textDescription}>Description:</div>
 
-      <GCText size={12}>{data.gunDescription}</GCText>
-    </div>
+      <div className={classes.frame}>
```

Comment:
please, think about using accordion or some new library component here. it will be useful to have it in common